setup.cfg

```
@@ -8,8 +8,6 @@ classifiers =
     Operating System :: OS Independent
     Programming Language :: Python
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
     Programming Language :: Python :: 3.12
```

```suggestion
    Programming Language :: Python :: 3.12
    Programming Language :: Python :: 3.13
```
Think we can add this with the CI running on 3.13rc2 now